modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Overview/index.js

```diff
@@ -34,73 +34,71 @@ const Overview = ({project, subscriptionGroups}) => {
 		SUBSCRIPTIONS_STATUS.future,
 	]);
 
+	const [allSubscriptions, setAllSubscriptions] = useState([]);
+
+	const [
+		subscriptionGroupsWithSubscriptions,
+		setSubscriptionGroupsWithSubscriptions,
+	] = useState([]);
+
```

Comment:
```suggestion
	const [subscriptionsGroups, setSubscriptionsGroups] = useState([]);

```

Let's reduce the number of states, I saw that the allSubscription state is only used in useEffect